dags/hindawi/xml_extractors.py

```diff
@@ -0,0 +1,71 @@
+import re
+import xml.etree.ElementTree as ET
+
+from common.parsing.xml_extractors import (
+    RequiredFieldNotFoundExtractionError,
+    TextExtractor,
+)
+
+
+class HindawiTextExtractor(TextExtractor):
+    def _get_none_value(self, value):
+        if value:
+            return value
+        return ""
+
+    def _get_extracted_value(self, node):
+        if self.required and node is None:
+            raise RequiredFieldNotFoundExtractionError(self.source)
+        if node is None:
+            return self.default_value
+        pattern = re.compile(r"[\n\t]*")
```

Comment:
maybe you can add a better naming what is this pattern

Reply:
This fix has to go to another branch, because this PR is for IOP, not hindawi publisher